src/connectors/refinement-list/__tests__/connectRefinementList-test.ts

```diff
@@ -2589,6 +2589,119 @@ See documentation: https://www.algolia.com/doc/api-reference/widgets/refinement-
         })
       );
     });
+
+    describe('facetOrdering', () => {
+      const resultsViaFacetOrdering = [
+        {
+          count: 66,
+          highlighted: 'Microsoft',
+          isRefined: false,
+          label: 'Microsoft',
+          value: 'Microsoft',
+        },
+        {
+          count: 88,
+          highlighted: 'Apple',
+          isRefined: true,
+          label: 'Apple',
+          value: 'Apple',
+        },
+        {
+          count: 44,
+          highlighted: 'Samsung',
+          isRefined: true,
+          label: 'Samsung',
+          value: 'Samsung',
+        },
+      ];
+      const resultsViaSortBy = [
+        {
+          count: 88,
+          highlighted: 'Apple',
+          isRefined: true,
+          label: 'Apple',
+          value: 'Apple',
+        },
+        {
+          count: 44,
+          highlighted: 'Samsung',
+          isRefined: true,
+          label: 'Samsung',
+          value: 'Samsung',
+        },
+        {
+          count: 66,
+          highlighted: 'Microsoft',
+          isRefined: false,
+          label: 'Microsoft',
+          value: 'Microsoft',
+        },
+      ];
+
+      test.each`
+        facetOrderingInResult | sortBy           | expected
+        ${true}               | ${undefined}     | ${resultsViaFacetOrdering}
+        ${false}              | ${undefined}     | ${resultsViaSortBy}
+        ${true}               | ${['isRefined']} | ${resultsViaSortBy}
+        ${false}              | ${['isRefined']} | ${resultsViaSortBy}
```

Comment:
I think it's be worth picking a `sortBy` input that yields a different result than the default value (same for `connectMenu-test.ts` and `connectHierarhicalMenu-test.ts`). Just having `resultsViaSortByDefaultValue` and `resultsViaSortByIsRefined`. 

The rational: right now, I could (accidentally) change the implementation to ignore `sortBy` prop and this test would still pass. 

Another test is still going to break, so I don't consider this a blocker, just a safer testing strategy. Not a blocker for this PR.